spec/examples/asciibib.adoc

```diff
@@ -16,23 +16,7 @@ title.language:: fr
 title.script:: Latn
 title.format:: text/plain
 type:: standard
-docid::
-docid.type:: ISO
-docid.id:: TC211
-docid::
-docid.type:: isbn
-docid.id:: ISBN
-docid::
-docid.type:: lccn
-docid.id:: LCCN
-docid::
-docid.type:: issn
-docid.scope:: series
-docid.id:: ISSN
-docid::
-docid.type:: URN
-docid.id:: urn:iso:std:iso:123:stage-90.93:ed-3:en,fr
-docnumber:: 123456
+[docid:: {type: "ISO", id: "TC211"}, docid:: {type: "isbn", id: "ISBN"}, docid:: {type: "lccn", id: "LCCN"}, docid:: {type: "issn", scope: "series", id: "ISSN"}, docid:: {type: "URN", id: "urn:iso:std:iso:123:stage-90.93:ed-3:en,fr"}]docnumber:: 123456
```

Comment:
This isn't legal Asciidoc, and the point of all this is that the asciidoc is processed more leniently, meaning that there should be a `docid:: XYZ` instance tested.